tidb-cloud/use-chat2query-api.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Get Started with Chat2Query API
+summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL statements using AI by providing instructions.
+---
+
+# Get Started with Chat2Query API
+
+TiDB Cloud provides the Chat2Query API, a RESTful interface that allows you to generate and execute SQL statements using AI by providing instructions. Then, the API returns the query results for you.
+
+Chat2Query API can only be accessed through HTTPS, ensuring that all data transmitted over the network is encrypted using TLS.
+
+> **Note:**
+>
+> Chat2Query API is only available for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
+
+## Before you begin
+
+Before using the Chat2Query API, make sure that you have created a [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) cluster. If you do not have one, follow the steps in [Create a cluster](/tidb-cloud/create-tidb-cluster.md) to create one.
```

Comment:
Also need to make sure that https://docs.pingcap.com/tidbcloud/explore-data-with-chat2query#enable-or-disable-ai-to-generate-sql-queries is enabled